topics/transcriptomics/metadata.yaml

```diff
@@ -57,6 +57,8 @@ material:
     hands_on: yes
     slides: no
     workflows: yes
+    tags:
```

Comment:
Most of our tutorials are for eukaryotes and I'm not sure its a good idea to add this particular label to all our tutorials.
I added one for prokaryotes, which imho makes more sense.